jwst/cube_build/ifu_cube.py

```diff
@@ -728,46 +559,54 @@ def build_ifucube(self):
 #                        else:
 #                            print('calling old cube_cloud')
                         cube_cloud.match_det2cube_msm(self.naxis1, self.naxis2, self.naxis3,
-                                                      self.Cdelt1, self.Cdelt2, self.Cdelt3,
-                                                      self.rois, self.roiw, self.weight_power,
+                                                      self.cdelt1, self.cdelt2,
+                                                      self.cdelt3_normal,
                                                       self.xcenters, self.ycenters, self.zcoord,
                                                       self.spaxel_flux,
                                                       self.spaxel_weight,
                                                       self.spaxel_iflux,
                                                       flux,
-                                                      coord1, coord2, wave)
+                                                      coord1, coord2, wave,
+                                                      rois_pixel, roiw_pixel, weight_pixel,
+                                                      softrad_pixel)
 
 
                         t1 = time.time()
-                        log.info("Time match point cloud to ifucube = %.1f.s" % (t1 - t0,))
+                        log.info("Time to match file to ifucube = %.1f.s" % (t1 - t0,))
```

Comment:
Eventually we probably don't want this kind of message to be at the regular "info" level, but perhaps "debug" instead. We really don't need timing info printed out in all processing logs for normal runs.

Reply:
Ok this was a request from the MIRI team so I stuck it in. They wanted to keep track of what ws going on because it was taking so long. I think after some more efficiency improvements I can take this out.